src/s3ql/fsck.py

```diff
@@ -1112,15 +1112,18 @@ def parse_args(args):
     parser.add_storage_url()
     parser.add_compress()
 
-    parser.add_argument("--keep-cache", action="store_true", default=False,
-                      help="Do not purge locally cached files on exit.")
-    parser.add_argument("--batch", action="store_true", default=False,
-                      help="If user input is required, exit without prompting.")
-    parser.add_argument("--force", action="store_true", default=False,
-                      help="Force checking even if file system is marked clean.")
-    parser.add_argument("--force-remote", action="store_true", default=False,
-                      help="Force use of remote metadata even when this would "
-                        "likely result in data loss.")
+    parser.add_argument("--keep-cache", action="store_true",
```

Comment:
Instead of making all these changes to every single `add_argument()` call, I think it's better to modify the `add_argument` function itself. You can derive the environment variable name from the parameter name and can leave all the add_argument() calls unchanged.

Reply:
hi, just did that! hope it's how you saw it being done